standards-catalogue/.schema/standards.json

```diff
@@ -0,0 +1,120 @@
+    "topics",
+    "license_id"
+  ],
+  "properties": {
+    "identifier": {
+      "$comment": "The unique identifier assigned to this standard",
+      "type": "string"
+    },
+    "name": {
+      "$comment": "The title or name of the standard",
+      "type": "string",
+      "maxLength": 70
+    },
+    "contraction": {
+      "$comment": "A shorter way of referring to the standard, for instance via an acronym",
+      "type": "string",
+      "maxLength": 10
+    },
+    "specification": {
+      "$ref": "#/definitions/specification"
+    },
+    "maintainer": {
+      "$comment": "The name or identifier of the organisation that maintains the standard",
+      "type": "string"
+    },
+    "catalogue_status_events": {
+      "$comment": "The events that this standard's status has been through",
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/catalogue_status_event"
+      }
+    },
+    "topics": {
+      "$comment": "The overarching topics that this fits under",
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "$ref": "#/definitions/topic"
+      }
+    },
+    "description": {
+      "$comment": "The long-form description of the standard, for instance written using Markdown",
+      "type": "string"
+    },
+    "license_id": {
+      "$comment": "The SPDX license identifier for the standard #### TODO??",
+      "type": "string",
+      "pattern": "(Proprietary|Unknown|.+)",
+      "minLength": 1
+    }
+  },
+  "additionalProperties": false,
+  "definitions": {
+    "topic": {
+      "$comment": "The human-readable topic descriptions that this standard relates to ",
+      "type": "string"
+    },
+    "specification": {
+      "$comment": "A container object to store information about the specification that this standard is **bound** to",
+      "type": "object",
+      "required": [
+        "name",
+        "url",
+        "published"
+      ],
+      "properties": {
+        "name": {
+          "$comment": "A the name of the specification for the standard",
+          "type": "string",
+          "minLength": 1
+        },
+        "url": {
+          "$comment": "A link to a publicly accessible specification for the standard",
+          "type": "string",
+          "pattern": "^https?://.*"
+        },
+        "published": {
+          "$comment": "The date that the specification was published",
+          "type": "string",
+          "format": "date"
+        }
+      }
+    },
+    "catalogue_status_event": {
+      "$comment": "A change in status for a standard, for the purpose of the standards catalogue's workflow",
+      "type": "object",
+      "properties": {
+        "status": {
+          "$ref": "#/definitions/status"
+        },
+        "date": {
+          "$comment": "Date that the change in status occurred",
+          "type": "string",
+          "format": "date"
+        }
+      }
+    },
+    "status": {
+      "type": "string",
+      "enum": [
```

Comment:
Hmm, should we split that up in terms of the `statuses` datafile? Or tweak it to something like:

```yaml
standard:
  active:
    name: Endorsed
    styling: status-active
  review:
    name: In review
    styling: status-draft
guidance:
  draft:
    name: Draft
    styling: status-draft
  published:
    name: Published
    styling: status-active
```

Then it's clear at-a-glance what are available, and it's easier to manage the logic, too

Reply:
yeah I think having it like that in one file is good 👍 also makes it easier to see where the same styling has been used in different statuses

Reply:
Having it in a single file with distinct grouping looks fine to me. We might end up with duplication if we add e.g. case studies which are likely to have similar statuses to guidance but surely we can change the grouping name when we cross that brigde.

As per the status identifiers, could we use 'endorsed' instead of 'active'? I think having only one term will reduce cognitive load and if we want to have an 'active' status at some other point it will be free?